### PR TITLE
Call playbook dispatcher on repo

### PR DESCRIPTION
**Pull request by acosferreira**
implement playbook dispatcher scheduling
Create and upload template Playbook
Call playbooks.ExecuteDispatcher()
Update/Create UpdateRecord.DispatchRecord

### CODE REVIEW

**Review by biancarosa — APPROVED** [review]
LGTM, let's just wait until the next demo to merge

**Review by biancarosa — APPROVED** [review]
LGTM

File: pkg/commits/repobuilder.go [hunk 1]
```diff
@@ -149,6 +151,46 @@ func (rb *RepoBuilder) BuildUpdateRepo(ut *models.UpdateTransaction) (*models.Up
...
 
+	// FIXME - implement playbook dispatcher scheduling
+	// 1. Create template Playbook
+	// 2. Upload templated playbook
+	var remoteInfo playbooks.TemplateRemoteInfo
+	remoteInfo.RemoteURL = update.Repo.URL
+	remoteInfo.RemoteName = update.Repo.Commit.Name
+	remoteInfo.ContentURL = update.Repo.URL
+	remoteInfo.UpdateTransaction = int(update.ID)
+	repoURL, err = playbooks.WriteTemplate(remoteInfo)
```
Comment (maxamillion) [inline pkg/commits/repobuilder.go:162]: Can we call `repoURL` here `playbookURL` or something similar, that variable name confused me. :)

**Review by maxamillion — APPROVED** [review]
LGTM, merging

**Pull request merged**